pic-build

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+this_dir=`dirname $0`
+build_dir=".build"
+
+help()
+{
+    echo "Build new binaries for a PIConGPU input set"
+    echo ""
+    echo "Creates or updates the binaries in an input set. This step needs to"
+    echo "be performed every time a .param file is changed."
```

Comment:
we don't mention them here to not overcomplicate things.
users should not interact with those.

in principle *anything* inside `include/` triggers a re-compile need (our check in `tbg` will remind them before submission as a safety net).